### PR TITLE
docs: add 8.0.1 release notes

### PR DESCRIPTION
**Pull request by dedemorton**

### CODE REVIEW

**Review by marclop — APPROVED** [review]
LGTM

File: changelogs/8.0.asciidoc [hunk 1]
```diff
@@ -3,13 +3,26 @@
...
 * <<release-notes-8.0.0-rc2>>
 * <<release-notes-8.0.0-rc1>>
 * <<release-notes-8.0.0-beta1>>
 * <<release-notes-8.0.0-alpha2>>
 * <<release-notes-8.0.0-alpha1>>
 
+[float]
+[[release-notes-8.0.1]]
+=== APM version 8.0.1
+
```
Comment (dedemorton) [inline changelogs/8.0.asciidoc:17]:
```suggestion
=== APM version 8.0.1

coming::[8.0.1]

```

File: changelogs/8.0.asciidoc [hunk 2]
```diff
@@ -3,13 +3,27 @@
...
+coming::[8.0.1]
```
Comment (dedemorton) [inline changelogs/8.0.asciidoc:18]:
```suggestion
coming::[8.0.1]

```

Comment (dedemorton) [reply changelogs/8.0.asciidoc:18]: Add empty line

**Pull request merged**